tests/sklearn/test_sklearn_models.py

```diff
@@ -1131,10 +1130,13 @@ def check_load_fitted_sklearn_linear_models(model_class, n_bits, x, y):
     y_pred_simulate = concrete_model.predict(x, fhe="simulate")
     y_pred_simulate_loaded = loaded_concrete_model.predict(x, fhe="simulate")
 
-    assert numpy.isclose(y_pred_simulate, y_pred_simulate_loaded).all(), (
-        "Simulated predictions from the initial model do not match the ones made from the "
-        "loaded one."
-    )
+    try:
+        check_float_array_equal(y_pred_simulate, y_pred_simulate_loaded)
```

Comment:
if you really want to give a proper message, it's better to add an `error_message` parameter in `check_float_array_equal` and pass it through that + append it to the initial error message instead of try except like this